.travis.yml

```diff
@@ -51,6 +51,50 @@ matrix:
             - zlib1g-dev
             - libgnutls28-dev
             - libonig-dev
+    # Adding jobs for ppc64le architecture
+    - name: "CONFIG_ARG=\"--with-tls=gnutls --with-regex=posix --with-migemo\""
+      os: linux
+      compiler: gcc
+      arch: ppc64le
+      env: CONFIG_ARG="--with-tls=gnutls --with-regex=posix --with-migemo"
```

Comment:
I think it's better to the purpose that build option sets follow distro package configuration ([debian] and [fedora]), but not copy x86_64 settings.

[debian]: https://salsa.debian.org/debian/jdim/-/blob/debian/sid/debian/rules#L12-16
[fedora]: https://src.fedoraproject.org/rpms/jd/blob/master/f/jd.spec#_133-144